README.md

```diff
@@ -29,36 +29,73 @@ Connect, execute a statement and handle results
 Make sure to use the configuration in the code that matches the version of Neo4j server that you run.
 
 ```go
-// Neo4j 4.0, defaults to no TLS therefore use bolt:// or neo4j://
-// Neo4j 3.5, defaults to self-signed cetificates, TLS on, therefore use bolt+ssc:// or neo4j+ssc://
-dbUri := "neo4j://localhost:7687"
-driver, err := neo4j.NewDriver(dbUri, neo4j.BasicAuth("username", "password", ""))
+package sample
+
+import (
+    "fmt"
+    "github.com/neo4j/neo4j-go-driver/v4/neo4j"
+    "io"
+    "log"
+)
+
+func example() error {
+    // Neo4j 4.0, defaults to no TLS therefore use bolt:// or neo4j://
+    // Neo4j 3.5, defaults to self-signed cetificates, TLS on, therefore use bolt+ssc:// or neo4j+ssc://
+    dbUri := "neo4j://localhost:7687"
+    driver, err := neo4j.NewDriver(dbUri, neo4j.BasicAuth("username", "password", ""))
+    if err != nil {
+        return err
+    }
+    // Handle driver lifetime based on your application lifetime requirements  driver's lifetime is usually
+    // bound by the application lifetime, which usually implies one driver instance per application
+    defer unsafeClose(driver)
+    
+    // Sessions are short-lived, cheap to create and NOT thread safe. Typically create one or more sessions
+    // per request in your web application. Make sure to call Close on the session when done.
+    // For multi-database support, set sessionConfig.DatabaseName to requested database
+    // Session config will default to write mode, if only reads are to be used configure session for
+    // read mode.
+    session := driver.NewSession(neo4j.SessionConfig{})
+    defer unsafeClose(session)
+    
+    result, err := session.WriteTransaction(createItem)
+    if err != nil {
+        return err
+    }
+    fmt.Printf("%v\n", result)
```

Comment:
Show the ugly casting needed?

Reply:
there are already some casts in `createItem`. Shall we show more of them? ^^

Reply:
Nah, maybe overkill... Meant the casting of result to *Item to make it a bit more realistic